keps/sig-auth/2843-request-ctx-in-token-review/README.md

```diff
@@ -0,0 +1,109 @@
+# KEP-2843: Forward Additional Request Metadata in TokenReviews
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Test Plan](#test-plan)
+  - [Graduation Criteria](#graduation-criteria)
+  - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+  - [Version Skew Strategy](#version-skew-strategy)
+<!-- /toc -->
+
+## Summary
+
+This KEP proposes enhancing the TokenReview API to convey additional request
```

Comment:
would you narrowly add this to TokenReview, or also to the authz and admission review objects?

Reply:
Discussed below.